cassandra/lib/JSONSchema.js

```diff
@@ -105,7 +105,12 @@ class JSONSchema {
      */
     static cassandraStringTypeOrDefault(type, val) {
         const stringTypes = [ 'text', 'varchar', 'ascii' ];
-        return stringTypes.includes(type) ? val.toString() : val;
+
+        if (stringTypes.includes(type)) {
+            return Utils.isObject(val) ? JSON.stringify(val) : val.toString();
```

Comment:
What if the **val** is null or undefined? Or what if the **val** does not has **toString** function (primitive type)? Or this situation is impossible?